phpunit/bootstrap.php

```diff
@@ -23,8 +23,8 @@
  * Manually load the plugin being tested.
  */
 function _manually_load_plugin() {
-	require dirname( dirname( __FILE__ ) ) . '/wxr-generator.php';
-	require dirname( dirname( __FILE__ ) ) . '/lib/class-buffer-writer.php';
+	require_once dirname( dirname( __FILE__ ) ) . '/wxr-generator.php';
```

Comment:
Without this, I kept getting:

```
Fatal error: Cannot declare class WXR_Generator\Buffer_Writer, because the name is already in use in /Users/vishnugopal/Developer/Automattic/wxr-generator/lib/class-buffer-writer.php on line 7
```